docs/koji.md

```diff
@@ -73,6 +73,8 @@ The docker map has these entries:
 - `repositories` (string list): docker pull specifications for the name:version-release image in the docker registry (or in Crane, if Pulp/Crane integration is used), by tag and by digests (there may be multiple digests, e.g. v2 schema 1 and v2 schema 2)
 - `config` (map): the [v2 schema 2 'config' object](https://docs.docker.com/registry/spec/manifest-v2-2/#image-manifest-field-descriptions) but with the 'container_config' entry removed
 - `tags` (string list): the image tags (i.e. the part after the ":") applied to this image when it was tagged and pushed
+- `layer_sizes` (map list): the image layer uncompressed sizes, the oldest layer first (the size information comes from docker history command),
```

Comment:
`layer_sizes` is a list

Reply:
A list of maps, i.e. a map list. We use that elsewhere e.g. "string list".

Reply:
This sounds a bit confusing to me, maybe we should expand this to 'a list of layers, each represented by a map of layer ID and its size`?

also `map` is not Pythonic, wouldn't it be better to use `dict` instead?

Reply:
Technically we're talking about JSON here, not Python. I used "map" originally following the content generator metadata format description here: https://docs.pagure.org/koji/content_generator_metadata/

How about we follow the existing style in this PR, but have a new PR to switch to something more clear? We can talk about ways of expressing it better in that new PR. One idea is to mix Python typing hints like "List[Mapping]" with JSON schema type names, giving something like "array[object]".

Thinking about this, though, makes me wonder if we have the format right to begin with. Perhaps we should name the fields like this?:
```
"layer_sizes": [
  {
    "diff_id": "sha256:012345...",
    "size": 12345
  },
  {
    "diff_id": "sha256:123456...",
    size: 23456
  }
]
```

Reply:
>Technically we're talking about JSON here, not Python

Oh, right. Agree

> have a new PR to switch to something more clear

Good idea, other fields might need some clarification too.

>Perhaps we should name the fields like this?

its good idea in case some additional fields would be added - but I can't think of any. Making `diff_id` the key in the dict sounds fine to me